README.md

```diff
@@ -119,6 +119,29 @@ The project is set up with [ESLint](https://eslint.org/) for JavaScript/TypeScri
 
 To ease development, we highly recommend enabling ESLint within your editor so you can see issues in real time. If you're using Visual Studio Code, you can use the [VS Code ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). For other editors, see [available ESLint integrations](https://eslint.org/docs/user-guide/integrations).
 
+## Internationalization (i18n)
```

Comment:
Best to add `yarn run build-i18n` in the README instructions above since it's a required step now

Reply:
In hindsight, maybe it's unnecessary for webpack to require `i18n.json` to exist (assuming everything defaults properly without it) *for development*, but I think it's important to ensure it exists for prod builds